evap/results/tests/test_tools.py

```diff
@@ -151,10 +157,17 @@ def test_average_deviation(self):
         mommy.make(RatingAnswerCounter, question=question_likert, contribution=contribution1, answer=5, count=4)
         mommy.make(RatingAnswerCounter, question=question_likert, contribution=general_contribution, answer=5, count=3)
 
-        __, deviation = calculate_average_grades_and_deviation(course)
+        # contribution1: 0.4 * (1, 0, 1, 0, 0) + 0.6 * (0, 0, 4, 0, 4) = (0.4, 0, 2.8, 0, 2.4) = (0.0714285714, 0, 0.5, 0, 0.4285714286)
+        # contribution2: (0, 2, 0, 2, 0) = (0, 0.5, 0, 0.5, 0)
+        # contributions: (0.0714285714, 0.5, 0.5, 0.5, 0.4285714286) = (0.0357142857, 0.25, 0.25, 0.25, 0.2142857143)
+
+        # course_non_grade: (0, 0, 0, 0, 3) = (0, 0, 0, 0, 1)
 
-        total_likert_dev = settings.CONTRIBUTION_PERCENTAGE * 1 + (1 - settings.CONTRIBUTION_PERCENTAGE) * 0
-        total_grade_dev = 1
-        total_dev = settings.GRADE_PERCENTAGE * total_grade_dev + (1 - settings.GRADE_PERCENTAGE) * total_likert_dev
+        # total: 0.375 * (0.0357142857, 0.25, 0.25, 0.25, 0.2142857143) + 0.625 * (0, 0, 0, 0, 1) = (0.0133928571, 0.09375, 0.09375, 0.09375, 0.7053571429)
```

Comment:
this is a comment to explain where the magic numbers in the test come from if something needs to be changed in the future, i can remove them - but that makes it really hard to follow.

Reply:
okay, i see it now. i think the comment is rather hard to follow. e.g. the equal signs are not correct, maybe use the counts_to_distribution function (or whatever it will be called) (in case you actually create that function)?

Reply:
yes, this was based on the assumption, that looking at it from a final distribution perspective the actual numbers are irrelevant and a distribution is only defined by proportions, so all the values are equal. i can add the function there somehow.